### DISCUSSION TITLE
首页推荐视频不感兴趣

### DISCUSSION THREAD

**Discussion by 125Justin** [the1812/Bilibili-Evolved#4820] [提问 / Q&A]
原版网页端首页推荐视频可以点击不感兴趣或者是不看此UP主，用了插件后就没有这个功能了，还是说隐藏在设置里。
![屏幕截图 2024-07-10 191643](https://github.com/the1812/Bilibili-Evolved/assets/72070863/28a91194-9bfc-40e6-b48a-5189e3001c42)

**Comment by the1812** [comment]
没这个功能
